### DISCUSSION TITLE
v1.4.1

### DISCUSSION THREAD

**Discussion by DogLooksGood** [meow-edit/meow#200] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/meow-edit/meow/releases/tag/v1.4.1'>v1.4.1</a>.</em>

**Comment by ChauhanT** [comment]
:( I'd love to try meow but I only have emacs 27.1 atm. Any chance it will be supported in the future?

**Reply by DogLooksGood** [reply]
I'd like to make it support  26.3 at least.

https://github.com/meow-edit/meow/issues/178

**Reply by DogLooksGood** [reply]
Hey, I surprised to find meow works on Emacs 27.1. - -#

**Reply by ChauhanT** [reply]
I tried to install in 27.1 but melpa says it is incompatible and doesn't let me install.

EDIT: Removed unimportant details.

**Reply by DogLooksGood** [reply]
Ohh, I see. I'll update the requirement, but you still have to wait melpa to build this package.

**Reply by ChauhanT** [reply]
Awesome, thank you. Can't wait to try it !

**Reply by ChauhanT** [reply]
Can confirm that it works. Thanks!
